pulsar-broker/src/main/java/com/yahoo/pulsar/broker/admin/Clusters.java

```diff
@@ -203,24 +206,23 @@ public void deleteCluster(@PathParam("cluster") String cluster) {
 
     @GET
     @Path("/{cluster}/namespaceIsolationPolicies")
-    @ApiOperation(value = "GGet the namespace isolation policies assigned in the cluster", response = NamespaceIsolationData.class, responseContainer = "Map")
+    @ApiOperation(value = "Get the namespace isolation policies assigned in the cluster", response = NamespaceIsolationData.class, responseContainer = "Map")
     @ApiResponses(value = { @ApiResponse(code = 403, message = "Don't have admin permission"),
             @ApiResponse(code = 404, message = "Cluster doesn't exist") })
     public Map<String, NamespaceIsolationData> getNamespaceIsolationPolicies(@PathParam("cluster") String cluster)
             throws Exception {
         validateSuperUserAccess();
-        validateClusterExists(cluster);
+        if (!clustersCache().get(path("clusters", cluster)).isPresent()) {
```

Comment:
`validateClusterExists()` throws 412 HTTP error if the cluster doesn't exist. But in this case we to return 404.